homeassistant/components/blink/config_flow.py

```diff
@@ -37,12 +48,8 @@ class BlinkConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
 
     def __init__(self):
         """Initialize the blink flow."""
-        self.blink = None
-        self.data = {
-            CONF_USERNAME: "",
-            CONF_PASSWORD: "",
-            "login_response": None,
-        }
+        self.auth = None
+        self.data = {CONF_USERNAME: "", CONF_PASSWORD: "", "device_id": DEVICE_ID}
```

Comment:
We don't need to make `data` an instance attribute. It's only used within a single step. Just make it a local variable.